src/resources/FundraisingPages.js

```diff
@@ -8,7 +8,7 @@ class FundraisingPages extends Resource {
         'activity',
         'posts'
       ],
-      creates: ['posts'],
+      creates: ['posts', 'transfers'],
```

Comment:
The endpoint is `/fundraising-pages/#/transfers` so I don't think we need a custom method, and I was anticipating using `classy.fundraisingPages.createTransfer(req.params.id, req.body)`